src/pages/RepoPage/CoverageOnboarding/ActivationBanner/ActivationBanner.tsx

```diff
@@ -0,0 +1,31 @@
+import { useParams } from 'react-router-dom'
+
+import { TrialStatuses, usePlanData } from 'services/account'
+import { isBasicPlan } from 'shared/utils/billing'
+
+import TrialEligibleBanner from './TrialEligibleBanner'
+
+function ActivationBanner() {
+  const { owner, provider } = useParams<{ owner: string; provider: string }>()
```

Comment:
Do you mind pulling out these types into an interface above the component, bit more consistent with the rest of the codebase, easier to reuse, and a bit cleaner.

```suggestion
interface URLParams {
  provider: string
  owner: string
}

function ActivationBanner() {
  const { owner, provider } = useParams<URLParams>()
```

Reply:
i don't mind! but tbh we've been using this across the codebase

Reply:
Just running through the codebase looks like there are 25 instances with `<{ ... }>` and 91 of `URLParams`, so I'm going to keep pushing for the use of `URLParams` for  greater consistency.